editor/animation/animation_track_editor.cpp

```diff
@@ -7972,11 +7981,12 @@ AnimationTrackEditor::AnimationTrackEditor() {
 	zoom->set_min(0.0);
 	zoom->set_max(2.0);
 	zoom->set_value(1.0);
-	zoom->set_custom_minimum_size(Size2(200, 0) * EDSCALE);
+	zoom->set_custom_minimum_size(Size2(128 * EDSCALE, 0));
```

Comment:
I think it could be set to an even lower value like 100 or 90.

Reply:
This is for the minimum slider width, i already have reduced it from 200 and allowed it to expand horizontally:

200px:
<img width="266" height="38" alt="image" src="https://github.com/user-attachments/assets/0142ffa8-eec7-407d-8d8b-391507822833" />

128px:
<img width="196" height="41" alt="image" src="https://github.com/user-attachments/assets/154efa02-7c20-483a-8561-4ddcf734e988" />

100px:
<img width="166" height="41" alt="image" src="https://github.com/user-attachments/assets/1e8d99a4-936e-409f-81a8-ce8b418e4353" />

At 100 i still can resize it without issue with dragging the slider or using the mouse wheel.